### PR TITLE
:running: Update the version of conversion-gen we are using

### PR DESCRIPTION
**Pull request by detiber**
**What this PR does / why we need it**:

Updates the version of conversion-gen to ensure that we get the same comments whether run inside or outside of the GOPATH

### CODE REVIEW

File: test/infrastructure/docker/hack/update-deps.sh [hunk 1]
```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/env bash
```
Comment (detiber) [inline test/infrastructure/docker/hack/update-deps.sh]: This was a second way to update deps in capd aside from the `make modules` target

File: test/infrastructure/docker/hack/verify-deps.sh [hunk 2]
```diff
@@ -1,50 +0,0 @@
-#!/usr/bin/env bash
```
Comment (detiber) [inline test/infrastructure/docker/hack/verify-deps.sh]: This was a second way to verify modules outside of the `make verify-modules` target

File: test/infrastructure/docker/hack/verify-all.sh [hunk 3]
```diff
@@ -37,13 +37,6 @@ failed=()
 outputs=()
 
 # run all verify scripts, optionally skipping any of them
-if [[ "${VERIFY_DEPS:-true}" == "true" ]]; then
-  echo "[*] Verifying deps..."
-  out=$(hack/verify-deps.sh 2>&1)
-  failure $? "verify-deps.sh" "${out}"
-  cd "${REPO_PATH}" || exit
-fi
-
```
Comment (detiber) [inline test/infrastructure/docker/hack/verify-all.sh]: Removes the secondary module verification in favor of the `make verify` target, which already includes the `verify-modules` target.

Comment (vincepri) [reply test/infrastructure/docker/hack/verify-all.sh]: Nice

**Pull request merged**